samples/dnn/person_reid.cpp

```diff
@@ -0,0 +1,255 @@
+//  This file is part of OpenCV project.
+//  It is subject to the license terms in the LICENSE file found in the top-level directory
+//  of this distribution and at http://opencv.org/license.html.
+// 
+// 
+//                           License Agreement
+//                 For Open Source Computer Vision Library
+// 
+//  Copyright 2020 Tencent Inc.  All rights reserved.
+//  Third party copyrights are property of their respective owners.
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+// 
+//        http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// 
+//  Author: Xing Sun <winfredsun@tencent.com>
+//          Feng Zheng <zfeng02@gmail.com>
+//          Xinyang Jiang <sevjiang@tencent.com>
+//          Fufu Yu <fufuyu@tencent.com>
+//          Enwei Zhang <miyozhang@tencent.com>
+
+/*
+You can download the converted pb model from https://https://www.dropbox.com/s/7831tlmz8lajhpy/youtu_reid_baseline_lite.onnx
+or use your own onnx model yourself.
+*/
+#include <iostream>
+#include <fstream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/dnn.hpp>
+
+using namespace cv;
+using namespace cv::dnn;
+
+const char* keys =
+"{help    h  |                 | show help message}"
+"{model   m  |                 | network model}"
+"{query_list q |               | list of query images}"
+"{gallery_list g |             | list of gallery images}"
+"{batch_size | 32              | batch size of each inference}"
+"{resize_h   | 256             | height of input image}"
+"{resize_w   | 128             | width of input image}"
```

Comment:
> of input image

IMHO, it doesn't look accurate.
This is about size of the input for model inference.

Reply:
hello, i have fixed all indentation mistakes, used references for parameters, and changed the description of resize_* to size of model.